inst/extdata/config_schema.json

```
@@ -0,0 +1,191 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$ref": "#/definitions/Epinow2",
+    "definitions": {
+        "Epinow2": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "job_id": {
+                    "type": "string",
+                    "format": "uuid"
+                },
+                "task_id": {
+                    "type": "string",
+                    "format": "uuid"
+                },
+                "as_of_date": {
+                    "type": "string",
+                    "format": "date"
+                },
+                "disease": {
+                    "type": "string"
+                },
+                "geo_value": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "geo_type": {
+                    "type": "string"
+                },
+                "parameters": {
+                    "$ref": "#/definitions/Parameters"
+                },
+                "data": {
+                    "$ref": "#/definitions/Data"
+                },
+                "seed": {
+                    "type": "integer"
+                },
+                "horizon": {
+                    "type": "integer"
+                },
+                "priors": {
+                    "$ref": "#/definitions/Priors"
+                },
+                "sampler_opts": {
+                    "$ref": "#/definitions/SamplerOpts"
+                }
+            },
+            "required": [
+                "as_of_date",
```

```suggestion
                "as_of_date",
                "timeseries_end_date",
                "timeseries_length_weeks",
```

These are needed so that we can do things like:

* Change the number of weeks in the sliding window
* Kick off retrospective runs (e.g. where the as_of_date is much later than the timeseries_end_date)

---

What is the logical relationship between as_of_date here, and `report_date` below inside the data block? Do we need to enforce / validate this relationship somewhere?